ffi/derive/src/lib.rs

```diff
@@ -276,21 +280,23 @@ pub fn ffi_import(_attr: TokenStream, item: TokenStream) -> TokenStream {
 }
 
 fn is_opaque(input: &syn::DeriveInput) -> bool {
+    let repr = find_attr(&input.attrs, "repr");
```

Comment:
don't you think optimizing compiler would do that?

Reply:
I don't know actually. If it will be smart enough to understand, that `find_attr()` doesn't produce any side effects, I think. But to me it looks like moving this will be more logical too